nexus/reconfigurator/rendezvous/src/lib.rs

```diff
@@ -13,29 +13,88 @@ use nexus_db_queries::db::DataStore;
 use nexus_types::deployment::Blueprint;
 use nexus_types::deployment::BlueprintDatasetFilter;
 use nexus_types::inventory::Collection;
+use serde::Deserialize;
+use serde::Serialize;
 
+mod crucible_dataset;
 mod debug_dataset;
 
+pub use crucible_dataset::CrucibleDatasetsRendezvousStats;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+pub struct RendezvousStats {
+    pub crucible_dataset: CrucibleDatasetsRendezvousStats,
```

Comment:
non-blocking question: no stats for debug datasets?

Reply:
Added in #7401 :)